web/main_test.go

```diff
@@ -13,9 +13,7 @@ import (
 
 	"github.com/apex/log"
 	uuid "github.com/satori/go.uuid"
-	// This is a hardcoded variable <-- should be moved
 	"github.com/unee-t-ins/invite"
-	// END This is a hardcoded variable
```

Comment:
Best to leave this comment: it will help us in the future to list all the hardcoded variables that we need to be mindful of.

Reply:
added comments.